src/components/Info.jsx

```diff
@@ -1,8 +1,12 @@
-import React from 'react';
+import React from "react";
+
+let command;
+
+window.navigator.platform.match(/^mac/) ? command = 'Cmd+O' : command = 'Ctrl+Shift+O';
 
```

Comment:
You could clean this up to be
const command = window.navigator.platform.match(/^mac/) ? 'Cmd+O' : 'Ctrl+Shift+O';

Reply:
Oh my. Why am I in such a hurry?...